### PR TITLE
Minify call stack even more

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
Somewhat tangential to Stephen's work here where the stack is too long: https://github.com/Microsoft/vscode-azuretools/pull/488. I figured another improvement would be just to collect and post less information from the stack.

Check the test cases to see what the call stack would look like. It contains all the info I ever found useful, but of course let me know if it's missing something major. I kind of feel better saying what we _do_ want to include rather than trying to remove things we _don't_ want to include in the stack.

### CODE REVIEW

**Comment by EricJizbaMSFT** [comment]
ping @StephenWeatherford updated thoughts on this? Should have the function name now...

File: ui/src/parseError.ts [hunk 1]
```diff
@@ -159,29 +158,40 @@ function getCallstack(error: { stack?: string }): string | undefined {
...
-
-    // Trim each line, including getting rid of 'at'
-    result = result.replace(/^\s*(at\s)?\s*/mg, '');
-    result = result.replace(/\s+$/mg, '');
+    const minifiedLines: (string | undefined)[] = stack
+        .split(/(\r\n|\n)/g) // split by line ending
+        .map(l => {
+            let result: string = '';
+            // Get just the file name, line number and column number
+            const fileMatch: RegExpMatchArray | null = l.match(/[^\/\\\(\s]+\.(t|j)s:[0-9]+:[0-9]+/i);
```
Comment (StephenWeatherford) [inline ui/src/parseError.ts]: It would be very helpful if you put an example line into the comments

Comment (EricJizbaMSFT) [reply ui/src/parseError.ts]: Isn't that what unit tests are for? 😂

Comment (StephenWeatherford) [reply ui/src/parseError.ts]: No.  :-)

**Comment by StephenWeatherford** [comment]
> Apheasement accepted :-)

**Pull request merged**